ddtrace/internal/datadog/profiling/code_provenance.py

```diff
@@ -87,6 +87,15 @@ def __init__(self):
             if module.endswith(".py") or module_path.is_dir():
                 lib.paths.add(str(module_path))
 
+        # If the user installed their code like a library and is running it as
+        # the main package (python -m my_package), and they explicitly specified
+        # that that's the main package, make sure it shows up as "my code" in
+        # the UI. Do this by leaving the kind blank (but not deleting the
+        # library so we can still associate the library with its files)
+        _, _, main_package = gitmetadata.get_git_tags()
```

Comment:
(unrelated) I find it a bit weird that we have to retrieve this information via a git metadata utility module 🤔

Reply:
Looks like we get the value using envier which in theory lets you get the config from other sources? https://github.com/DataDog/dd-trace-py/blob/594a5019a389f5b2fbf2905474727ae05a05f9af/ddtrace/internal/gitmetadata.py#L30. Not sure if anybody sets this variably via anything other than an environment variable today, but I'd rather be consistent with the rest of the code and use this module.